gromacs/config.py

```diff
@@ -724,9 +724,18 @@ def set_gmxrc_environment(gmxrc):
         logger.warning("Failed to automatically set the Gromacs environment"
                        "from GMXRC=%r", gmxrc)
 
-#: Python list of all tool file names. Filled from values in the tool
-#: groups in the configuration file.
-load_tools = []
-for g in cfg.getlist('Gromacs', 'groups', sort=False):
-     load_tools.extend(cfg.getlist('Gromacs', g))
-del g
+
+def get_tools():
+    """ Get tool names from all configured groups.
+
+    :return: list of tool names
+
+    """
+    load_tools = []
+    for g in cfg.getlist('Gromacs', 'groups', sort=False):
+         load_tools.extend(cfg.getlist('Gromacs', g, sort=False))
+    return load_tools
+
+
+RELEASE = cfg.get('Gromacs', 'release')
+MAJOR_RELEASE = RELEASE.split('.')[0]
```

Comment:
What do you want to use the `MAJOR_RELEASE` for?

It's not a bad idea but at the moment, `release` is not actually required in the cfg file. At a minimum we should allow something like `5.1.*` or even `5.*.*` so that users don't have to declare every patch-level release of Gromacs.

Furthermore, if Gromacs changes their release numbering scheme then we have to change this, too.

For right now, put this in a try/except and if getting a `RELEASE` fails, just set everything to `None` (or to 4 or 5, depending on the guessing elsewhere?)